saskatoon/harvest/serializers.py

```diff
@@ -1,5 +1,5 @@
 from rest_framework import serializers
-from .models import Harvest, Property, Equipment, EquipmentType, RequestForParticipation
+from harvest.models import Harvest, Property, Equipment, EquipmentType, RequestForParticipation
```

Comment:
I think this will be the only change left

Reply:
Yes